docs/arcaflow/workflows/expressions.md

```diff
@@ -28,31 +70,256 @@ $.foo.bar
+
+For example, `2 != 2` results in `false`, and `"a" != "b"` results in `true`.
+
+### Greater Than
+
+The `>` inequality operator outputs `true` if the left side is greater than the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 > 3` would output the boolean `false`, and `4 > 3` would output `true`.
+
+### Less Than
+
+The `<` inequality operator outputs `true` if the left side is less than the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 < 3` would output the boolean `false`, and `1 < 2` would output `true`.
+
+### Greater Than or Equal To
+
+The `>=` inequality operator outputs `true` if the left side is greater than or equal to the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 >= 3` would output the boolean `true`, `3 >= 4` would output `false`, and `4 >= 3` would output `true`.
+
+### Less Than or Equal To
+
+The `<=` inequality operator outputs `true` if the left side is less than or equal to the right side, and `false` otherwise. The operator requires numerical or string input.
+
+For example, the expression `3 <= 3` would output the boolean `true`, `3 <= 4` would output `true`, and `4 <= 3` would output `false`.
+
+### Logical AND
+
+The `&&` operator returns `true` if both the left and right sides are `true`, and `false` otherwise. This operator requires boolean input.
+Note: There is no short-circuiting as it's currently implemented. Both the left and right are evaluated before the comparison takes place.
+
+All cases:
+| Left    | Right   | `&&`    |
+| ------- | ------- | ------- |
+| `true`  | `true`  | `true`  |
+| `true`  | `false` | `false` |
+| `false` | `true`  | `false` |
+| `false` | `false` | `false` |
+
+### Logical OR
+
+The `&&` operator returns `true` if either or both of the left and right sides are `true`, and `false` otherwise. It is a non-exclusive or operator. This operator requires boolean input.
+Note: There is no short-circuiting as it's currently implemented. Both the left and right are evaluated before the comparison takes place.
+
+All cases:
+| Left    | Right   | `\|\|`  |
+| ------- | ------- | ------- |
+| `true`  | `true`  | `true`  |
+| `true`  | `false` | `true`  |
+| `false` | `true`  | `true`  |
+| `false` | `false` | `false` |
+
+## Unary Operations
+
+These are operations that have one operator before the expressions.
+
+| Operator | Description        |
+| ---------|--------------------|
+| -        | [Negation](#Negation)           |
+| !        | [Logical complement](#logical-complement) |
+
+### Negation
+
+The negation operator negates the value from the input expression.
+The required format is a dash `-` before the expression.
```

Comment:
I think it's worth keeping the second sentence. I'd rather be explicit in case someone has tunnel vision.
I think from sounds better. But I added "numeric" as requested by Dave.